bmc/engine/check_settling_time.h

```diff
@@ -0,0 +1,170 @@
+/*
+ * check_settling_time.h
+ *
+ *  Created on: Nov 25, 2017
+ *      Author: Thiago Cavalcante
+ */
+
+#ifndef CHECK_SETTLING_TIME_H_
+#define CHECK_SETTLING_TIME_H_
+
+#include <Eigen/Dense>
+#include <Eigen/Core>
+#include <complex>
+#include <cstdlib>
+
+extern digital_system_state_space _controller;
+double new_state[LIMIT][LIMIT];
+
+using Eigen::MatrixXd;
+
+// Calculate the system's output
```

Comment:
Can you please use the doxygen standard to comment all functions?

Essentially, you need to describe the purpose of the function, its input arguments and return.

Please take a look at this file as an example: https://github.com/ssvlab/dsverifier/blob/master/bmc/core/fixed_point.h